tests/src/rules/order.js

```diff
@@ -3,7 +3,7 @@ import { test, getTSParsers } from '../utils'
 import { RuleTester } from 'eslint'
 
 const ruleTester = new RuleTester()
-    , rule = require('rules/order')
+    , rule = require('../../../src/rules/order')
```

Comment:
not sure why this is here; `npm run mocha path/to/file` will set up the NODE_PATH for you to point to `src`.
```suggestion
    , rule = require('rules/order')
```

Reply:
This is how I run tests:
```sh
npx mocha --compilers js:babel-register tests/src/rules/order.js
```
Without that change the `require` fails.

Reply:
Right, that's not a valid way to run tests in this repo - you need to use `npm run mocha path/to/file` instead.